website/docs/r/cwf_process.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: CWF Process"
+sidebar_current: "docs-rightscale-resource-cwf_process"
+description: |-
+  Create and maintain a rightscale CloudWorkFlow process.
+---
+
+# rightscale_cwf_process
+
+Use this resource to create or destroy RightScale [CloudWorkFlow processes](http://docs.rightscale.com/ss/reference/rcl/).
+
+Creating the CWF process will run it synchronously and return the output values (if any).
+
+Destroying a CWF process will remove it from the CWF Console table of executed processes. It WON'T undo any action that it could have executed nor would stop any running process.
+
+It is NOT possible to update a CWF process.
+
+## Example Usage
+
+This example CWF process will look for all servers whose name starts by "db-slave-" and will execute the specified RightScript on them,
+returning the number of servers that have been reached.
+
+```hcl
+resource "rightscale_cwf_process" "run_executable_by_prefix" {
+
+  parameters = [
+     { "kind" = "string"
+       "value" = "db-slave-" },
+     { "kind" = "string"
```

Comment:
missing `,`

Reply:
It's correct, TF maps don't separate key-values with comas, the above example has been tested and it's like this in [official examples](https://www.terraform.io/docs/configuration/variables.html#example)